pkg/analyzer/analyzers/asana/asana.go

```diff
@@ -25,19 +27,68 @@ type Analyzer struct {
 func (Analyzer) Type() analyzerpb.AnalyzerType { return analyzerpb.AnalyzerType_Asana }
 
 func (a Analyzer) Analyze(_ context.Context, credInfo map[string]string) (*analyzers.AnalyzerResult, error) {
-	_, err := AnalyzePermissions(a.Cfg, credInfo["key"])
+	key, ok := credInfo["key"]
+	if !ok {
+		return nil, errors.New("key not found in credInfo")
+	}
+
+	info, err := AnalyzePermissions(a.Cfg, key)
 	if err != nil {
 		return nil, err
 	}
-	return nil, fmt.Errorf("not implemented")
+
+	return secretInfoToAnalyzerResult(info), nil
+}
+
+func secretInfoToAnalyzerResult(info *SecretInfo) *analyzers.AnalyzerResult {
+	if info == nil {
+		return nil
+	}
+
+	result := analyzers.AnalyzerResult{}
+
+	// resoures/permission setup
+	permissions := allPermissions()
+	userResource := analyzers.Resource{
+		Name:               info.Data.Name,
+		FullyQualifiedName: info.Data.ID,
+		Type:               "user",
+		Metadata: map[string]any{
+			"email": info.Data.Email,
+			"type":  info.Data.Type,
+		},
+	}
+
+	for _, perm := range permissions {
+		binding := analyzers.Binding{
+			Resource:   userResource,
+			Permission: perm,
+		}
+		result.Bindings = append(result.Bindings, binding)
+	}
```

Comment:
**Nit:** I think we could use `analyzers.BindAllPermissions` here

Reply:
Thanks for pointing it our. Code updated